interp/constrintern.ml

```diff
@@ -1369,7 +1369,7 @@ let sort_fields ~complete loc fields completer =
                (* the order does not matter as we sort them next,
                   List.rev_* is just for efficiency *)
                let remaining_fields =
-                 let complete_field (idx, _field_ref) = (idx, completer idx) in
+                 let complete_field (idx, _field_ref) = (idx, completer idx _field_ref base_constructor) in
```

Comment:
The leading underscore should be removed.

It's a convention to use a leading underscore when you want to indicate that something is unused but you still want it to have a name. We don't do it for all unused variables but no used variable should start with an underscore.